hw-05/genre_repositories.py

```diff
@@ -0,0 +1,20 @@
+from database_handler import execute_query
+
+
+def get_most_popular_city_by_genre(genre_name):
```

Comment:
some genre could be popular in a few cities, update your code based on this

for example Rock And Roll

Reply:
Thanks, I fixed this mistake.